server/apps/immich/src/api-v1/album/album.service.spec.ts

```diff
@@ -0,0 +1,414 @@
+import { AlbumService } from './album.service';
+import { IAlbumRepository } from './album-repository';
+import { AuthUserDto } from '../../decorators/auth-user.decorator';
+import { BadRequestException, NotFoundException, ForbiddenException } from '@nestjs/common';
+import { AlbumEntity } from '@app/database/entities/album.entity';
+import { Album } from './response-dto/album';
+
```

Comment:
I am learning a lot of about testing from reading these - Thank you very much for that!

Reply:
My pleasure. Next is trying TDD (Red - Green - Refactor) 😉